src/handlers/ipcHandlers/facilityHandler.js

```diff
@@ -21,6 +22,13 @@ const getFacilitiesByRegion = (e, arg) => {
     });
 }
 
+const getFacilitiesByState = (e,arg) => {
```

Comment:
What are `e` and `arg`?

Reply:
event and arguments. I think thats the convention for those names in js.

Reply:
Hm, they don't very much explain what we're sending though even if they follow the common convention. Like, without looking into the function I would not know what they are. For example, is `arg` the ID?

What do you think of using more explanatory names, regardless of the common convention, to create our own convention?

cc @extremebip

Reply:
I guess what you could do is specify the args but that woudl totally break convention if im being honest since event handlers are suppose to be independent and streamlined in such a way. The event object is just an event object tho i dont think theres a way to rename that to be clearer.

Reply:
I think if we establish our own conventions that's fine. As long as the team can communicate. So the questions are:

1. Is it better? As a developer not wholly focused on JS, I'm not entirely familiar with how this is usually set up
2. Should we do it?

Reply:
Yeah i think we should discuss this before making changes.

Generally handler functions never specify their args in js so theres that.

Reply:
Then for now we'll follow the common convention. Resolving.